backend/src/bootstrap.ts

```diff
@@ -0,0 +1,43 @@
+import ora from "ora";
+import nodemailer from "nodemailer";
+import { getSmtpHost, getSmtpPort } from "./config";
+import { logger } from "./utils/logging";
+import mongoose from "mongoose";
+import { redisClient } from "./services/RedisService";
+
+type BootstrapOpt = {
+  transporter: nodemailer.Transporter;
+};
+
+export const bootstrap = async ({ transporter }: BootstrapOpt) => {
+  const spinner = ora().start();
```

Comment:
I don't think this spinner is necessary here. Please only add dependencies for must have items so we can reduce # of dependency.  I think we can just use our logger for this instead

Reply:
The issue with logger is for each message there is a stack trace too from where it gets fired etc. 

So putting these messages there maynot give users a good experience when seeing it. This would give a elegant experience and i wanna add more things to it later. Also ora is a tiny library too maintained by a great dev, used by many company. 

If i replace this with logger it would look a bit awful on booting up for customers